guides/advanced_guides/README.md

```diff
@@ -1,6 +1,6 @@
 # Advanced guides
 
-In the advanced guides you will find how to tune your search API and customize it:
+The advanced guides cover how to tune your search API and customize it:
```

Comment:
Because not all advanced guides cover how to tune. Some only explain how MeiliSearch works.
```suggestion
These advanced guides cover how to tune your search API and customize it:
```